src/store/selectors/farmExtensionSelectors.js

```diff
@@ -0,0 +1,324 @@
+  return farmData.motivations
+}
+
+export function getOperationTypes (group) {
+  const farmData = getFarmData(group)
+  if (!farmData || !farmData.types) return []
+  return farmData.types
+}
+
+export function getOrganizationId (group) { // country specific business id
+  const farmData = getFarmData(group)
+  if (!farmData || !farmData.organization_id) return null
+  return farmData.organization_id
+}
+
+export function getOrganizationName (group) {
+  const farmData = getFarmData(group)
+  if (!farmData || !farmData.organization) return null
+  return farmData.organization
+}
+
+export function getLandOwnership (group) {
+  const farmData = getFarmData(group)
+  if (!farmData || !farmData.land_other) return []
+  return farmData.land_other
+}
+
+export function getLandOwnershipDetail (group) {
+  const farmData = getFarmData(group)
+  if (!farmData || !farmData.land_other_detail) return {}
+  return farmData.land_other_detail
+}
+
+export function getBio (group) {
+  const farmData = getFarmData(group)
+  if (!farmData || !farmData.bio) return null
+  return farmData.bio
+}
+
+export function getPreferredContactMethod (group) {
+  const farmData = getFarmData(group)
+  if (!farmData || !farmData.preferred) return null
+  return farmData.preferred
+}
+
+export function getProducerCommunityOutlineGeo (group) {
+  const farmData = getFarmData(group)
+  if (!farmData || !farmData.area_community) return null
+  return farmData.area_community
+}
+
+export function getProductDetail (group) {
+  const farmData = getFarmData(group)
+  if (!farmData || !farmData.product_detail) return []
+  return farmData.product_detail
+}
+
+export function getProductsValueAdded (group) {
+  const farmData = getFarmData(group)
+  if (!farmData || !farmData.products_value_added) return []
+  return farmData.products_value_added
+}
+
+export function getSchemaVersion (group) {
+  const farmData = getFarmData(group)
+  if (!farmData || !farmData.schema_version) return []
+  return farmData.schema_version
+}
+
+export function getSocialMedia (group) {
+  const farmData = getFarmData(group)
+  if (!farmData || !farmData.social) return null
+  return farmData.social
+}
+
+export function getAnimalTotalCount (group) {
+  const farmData = getFarmData(group)
+  if (!farmData || !farmData.animals_total) return null
+  return farmData.animals_total
+}
+
+export function getAnimalCountByType (group) {
+  const farmData = getFarmData(group)
+  if (!farmData || !farmData.animals_detail) return null
+  return farmData.animals_detail
+}
+
+export function getUnitPreference (group) {
+  const farmData = getFarmData(group)
+  if (!farmData || !farmData.units) return 'metric'
+  return farmData.units
+}
+
+/*
+  Hylo sub-profile specific selectors
+*/
+
+export function getHyloSpecificFarmSchemaData (farmData) {
+  if (!farmData || !farmData.flexible || !farmData.flexible.hylo) return {}
+  return farmData.flexible.hylo
+}
+
+export function getOpenToPublic (group) {
```

Comment:
note: might move some or all of these to the regular group.settings so we dont have to load the extension data to look this up